tests/pcap/src/handshake_message.rs

```diff
@@ -0,0 +1,178 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+use anyhow::*;
+use rtshark::Packet;
+use rtshark::RTSharkBuilder;
+use std::collections::HashSet;
+
+use crate::capture::*;
+
+#[derive(Debug, Clone, Default)]
+pub struct Message {
+    /// TLS handshake message type
+    pub message_type: u8,
+    /// Packet containing handshake message metadata
+    pub packet: Packet,
+    /// Ordered TCP payloads as hex encoded strings
```

Comment:
Would it be better to parse the hex encoding strings into a `Vec<u8>` instead? So you can actually operate on the data rather than the caller having to parse it?

Reply:
My thinking is that we don't know that the user wants bytes. They could just want to print it or compare it to another payload, in which case hex is fine or even arguably better. If they don't want hex, they can decode.

But I can switch it to Vec<u8> if you disagree.

Reply:
I think I still prefer `Vec`. Formatting into hex is pretty easy/niche. Processing the raw bytes seems like the more likely case.